core/src/main/java/jenkins/diagnostics/URICheckEncodingMonitor.java

```diff
@@ -43,7 +43,7 @@ public FormValidation doCheckURIEncoding(StaplerRequest request) throws IOExcept
 
         if (!expected.equals(value)) {
             String expectedHex = Util.toHexString(expected.getBytes(StandardCharsets.UTF_8));
-            String valueHex = Util.toHexString(value.getBytes(StandardCharsets.UTF_8));
+            String valueHex = value != null ? Util.toHexString(value.getBytes(StandardCharsets.UTF_8)) : "<empty string>";
```

Comment:
(Optional suggestion.) Seems inconsistent to me that when the "value" parameter is empty that `value` would be `null` but `valueHex` would be "empty string". The below is more consistent but it does not matter either way.

```suggestion
            String valueHex = value != null ? Util.toHexString(value.getBytes(StandardCharsets.UTF_8)) : null;
```